tests/widget/modal/widget-modal.login.test.js

```diff
@@ -19,4 +26,13 @@ test('Modal widget with login', async ({ page }) => {
   await clickButton('Sign In', '/authenticate');
 
   await verifyUserInfo(page, expect);
+
+  // Navigate to page without Widget instantiation
+  await navigate('widget');
+
+  // Refresh the page to clear out previous Widget references
+  await page.reload({ waitUntil: 'networkidle' });
+
+  // Test the config and user API without Widget references
+  await verifyUserInfo(page, expect);
```

Comment:
Navigate back to the basic page and ensure user information renders without widget instantiation.